src/main/java/com/taskadapter/redmineapi/RedmineManager.java

```diff
@@ -118,4 +118,7 @@ public void setOnBehalfOfUser(String loginName) {
         transport.setOnBehalfOfUser(loginName);
     }
 
+    public Transport getTransport() {
+        return transport;
+    }
 }
```

Comment:
I don't mind having this method, but it needs to be documented - what is the consumer expected to do with it? what are the limitations? etc

Reply:
It's just a getter that lets the developer access the underlying Transport. For the time being, I'm accessing the Transport by Java reflection (not ideal).

As I've showed in my initial PR comment, it allows to call any method of the Transport instance, which is useful if you want to be able to use this lib to call standard Redmine API endpoints *and* custom ones that have been developed for custom requirements.

What type of document would you expect? Javadoc? Markdown in the Readme or Wiki?